packages/build-tools/src/utils/project.ts

```diff
@@ -19,6 +19,16 @@ export async function setup<TJob extends Job>(ctx: BuildContext<TJob>): Promise<
     }
   });
 
+  await ctx.runBuildPhase(BuildPhase.READ_PACKAGE_JSON, async () => {
```

Comment:
I don't think we should add any more phases, especially very short ones. I would prefer to add that to the phase where we read app config and maybe rename that phase.

Also we will be adding `expo doctor` soo so that change will also land in the same build phase

Reply:
What would be the main reason for combining them? Does it cause any overhead?

Reply:
We had an offline discussion with @wkozyra95 and we think that having multiple build phases is fine. Can you move the phase so it's just before `READ_APP_CONFIG`?